flytekit/types/schema.py

```diff
@@ -299,7 +299,7 @@ def as_readonly(self) -> FlyteSchema:
 
 
 class FlyteSchemaTransformer(TypeTransformer[FlyteSchema]):
-    _SUPPORTED_TYPES: typing.Dict[type : SchemaType.SchemaColumn.SchemaColumnType] = {
+    _SUPPORTED_TYPES: typing.Dict[Any, SchemaType.SchemaColumn.SchemaColumnType] = {
```

Comment:
should this be `Type`

Reply:
I think here `Any` seems to be correct. `Type` is used to annotate class objects, but here I guess it should just be instance of some type, right?

Reply:
Ah wait, you are right. I miss understood the code.